src/main/java/core/mvc/asis/DispatcherServlet.java

```diff
@@ -1,80 +1,76 @@
 package core.mvc.asis;
 
+import core.mvc.HandlerMapping;
 import core.mvc.ModelAndView;
 import core.mvc.tobe.AnnotationHandlerMapping;
 import core.mvc.tobe.HandlerExecution;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.servlet.RequestDispatcher;
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Objects;
 
 @WebServlet(name = "dispatcher", urlPatterns = "/", loadOnStartup = 1)
 public class DispatcherServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
     private static final String CONTROLLER_PATH = "next.controller";
 
-    // TODO 추후 삭제
-    private static final String DEFAULT_REDIRECT_PREFIX = "redirect:";
-
     private static final Logger logger = LoggerFactory.getLogger(DispatcherServlet.class);
 
-    private AnnotationHandlerMapping mapping;
+    private static final List<HandlerMapping> handlerMappings = new ArrayList<>();
 
-    private RequestMapping rm;
+    private AnnotationHandlerMapping mapping;
+    private LegacyHandlerMapping rm;
 
     @Override
     public void init() throws ServletException {
         mapping = new AnnotationHandlerMapping(CONTROLLER_PATH);
         mapping.initialize();
 
         // TODO 추후 삭제
-        rm = new RequestMapping();
+        rm = new LegacyHandlerMapping();
         rm.initMapping();
+
+        handlerMappings.add(mapping);
+        handlerMappings.add(rm);
```

Comment:
모든 `HandlerMapping` 은 초기화 함수가 위에서만 호출되고 있네요.
초기화를 생성자에서 해주면 아래와 같이 좀 더 간결하게 되겠네요 :)
```suggestion
        handlerMappings.add(new AnnotationHandlerMapping(CONTROLLER_PATH));
        handlerMappings.add(new LegacyHandlerMapping());
```